src/plugins/template/tests/functional/op_reference/convert_like.cpp

```diff
@@ -109,6 +111,13 @@ INSTANTIATE_TEST_SUITE_P(
                       ov::element::bf16,
                       std::vector<uint8_t>{0, 10, 15, 20, 43, 56, 78, 99, 102, 130, 142},
                       std::vector<bfloat16>{0, 10, 15, 20, 43, 56, 78, 99, 102, 130, 142}),
+        ConvertParams(ConversionTypes::CONVERT_LIKE,
+                      ov::PartialShape{11},
+                      ov::element::f4e2m1,
+                      ov::element::bf16,
+                      // only one nibble got value others are be 0
+                      std::vector<float4_e2m1>{0.0f, -1.0f, -0.5f, 6.0f, -4.0f, 1.5f},
+                      std::vector<bfloat16>{0.0f, 0.0f, -1.0f, 0.0f, -0.5f, 0.0f, 6.0f, 0.0f, -4.0f, 0.0f, 1.5f}),
```

Comment:
?
```suggestion
                      std::vector<bfloat16>{0.0f, 0.0f, -1.0f, 0.0f, -0.5f, 0.0f, 6.0f, 0.0f, -4.0f, 0.0f, 1.5f, 0.0f}),
```

Reply:
It will be incorrect as will change output shape to 12 but input has got 11 elements.

Reply:
Could you share an example result/test for the same case but with the input shape {6} (equal to the number of float4 values)?